src/app/ui/mission/MissionWidget/MissionVerifyWallet.tsx

```
@@ -0,0 +1,226 @@
+
+  const getTranslationWithDiscordLink = (key: ParseKeys<'translation'>) => {
+    return (
+      <Trans<ParseKeys<'translation'>>
+        i18nKey={key}
+        components={[
+          <Link
+            href={DISCORD_URL}
+            target="_blank"
+            rel="noopener noreferrer"
+            sx={{ fontWeight: 'bold' }}
+          />,
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)